### PR TITLE
Chrony support

### PR DESCRIPTION
**Pull request by Serphentas**
This is to add support for Chrony (another NTP implementation). Some graphs can be seen below. Overall, this tries to provide the same insight given by `ntp-server` but using the output from `chronyc`.

The agent can be found here: https://github.com/librenms/librenms-agent/pull/345

graph_gps

graph_pps

graph_tracking

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [X] Have you followed our [code guidelines?](https://docs.librenms.org/Developing/Code-Guidelines/)
- [X] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

**Comment by Jellyfrog** [comment]
@Serphentas  Please check failed tests

File: includes/html/functions.inc.php [hunk 1]
```diff
@@ -1628,3 +1628,29 @@ function nfsen_live_dir($hostname)
         }
     }
 }
+
+/**
+ * Get the ZFS pools for a device... just requires the device ID
```
Comment (yrebrac) [inline includes/html/functions.inc.php:1633]: refers to ZFS

**Comment by Jellyfrog** [comment]
@Serphentas something left to do here? I fixed the tests for you

**Comment by Serphentas** [comment]
@Jellyfrog many thanks for the fix... which quite surprised me in its simplicity. Didn't have time to spare for debug recently, so we can roll with this yeah.

**Review by Jellyfrog — APPROVED** [review]

**Pull request merged**